stages/test/test_anaconda.py

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/python3
+
+import os.path
+
+import pytest
+
+import osbuild.meta
+
+
+@pytest.mark.parametrize(
+    "test_data,expected_err",
+    [
+        # BAD patterns that should not work
+        ({"kickstart-modules": []}, "[] is too short"),
+        ({"activatable-modules": []}, "[] is too short"),
+        ({"forbidden-modules": []}, "[] is too short"),
+        ({"optional-modules": []}, "[] is too short"),
+        # GOOD patterns that should work, starting with a fully empty one
+        ({}, ""),
+        ({"kickstart-modules": ["test"]}, ""),
+        ({"activatable-modules": ["test"]}, ""),
+        ({"forbidden-modules": ["test"]}, ""),
+        ({"optional-modules": ["test"]}, ""),
+    ],
+)
+def test_schema_validation_smoke(test_data, expected_err):
+    name = "org.osbuild.anaconda"
```

Comment:
I suspect moving a helper into testutil to get the schema might make sense now. I also copy/pasted this in my erofs test so maybe a single place is better (?). But not 100% convinced as it's just 3 lines (and dirname(__file__) is then much less obvious and more complicated) in the end so maybe fine